### PR TITLE
Add new Accepted tests in hertzbeat

### PR DESCRIPTION
**Pull request by bbelide2**
Adding 2 new Accepted tests in [hertzbeat](https://github.com/dromara/hertzbeat).

Both the tests are ID: details and logs are present in https://github.com/TestingResearchIllinois/idoft/issues/1148
Root cause and fix details are present in the PRs:

https://github.com/dromara/hertzbeat/pull/1373
https://github.com/dromara/hertzbeat/pull/1371

### CODE REVIEW

File: format_checker/forked-projects.json [hunk 1]
```diff
@@ -467,4 +468,4 @@
   "https://github.com/zeroturnaround/gradle-jrebel-plugin": "unforked",
   "https://github.com/zhangxd1989/spring-boot-cloud": "unforked",
   "https://github.com/zhurlik/gradle-jboss-modules-plugin": "unforked"
-}
```
Comment (darko-marinov) [inline format_checker/forked-projects.json:470]: What editor/IDE are you using to get this change automatically? At some point we may accept that change to stop dealing with unnecessary changes.

Comment (bbelide2) [reply format_checker/forked-projects.json:470]: I usually use vs code but for this change I directly made in GitHub by editing the file

Comment (darko-marinov) [reply format_checker/forked-projects.json:470]: Can you test on another file if the GitHub editor automatically removes or adds newline at the end of the file?

Comment (bbelide2) [reply format_checker/forked-projects.json:470]: I edited another file requirements.txt. I just removed 1 and added 1 in the last line. That line is shown in the diff despite being exactly same.

Comment (darko-marinov) [reply format_checker/forked-projects.json:470]: The line is not the same if it's in the diff. Can you search online about reports of GitHub editor adding/removing the newline character? Can that behavior be customized?

Comment (bbelide2) [reply format_checker/forked-projects.json:470]: In "Files changed" tab > settings > I checked the box "Hide whitespaces" and all the unnecessary diff is removed. Only the changed lines are shown.

Comment (bbelide2) [reply format_checker/forked-projects.json:470]: I searched online and in github settings, but did not find any setting to control new lines in the gitghub web editor. But there are options to manage these configurations at IDE level like in VS code. Also, there is an option in GitHub to edit a file in github.dev which opens the repo in an online vs code editor.

Comment (bbelide2) [reply format_checker/forked-projects.json:470]: In this PR, I was not able to reporoduce the exact diff in new branches, I may have inadvertantly changed some space

Comment (darko-marinov) [reply format_checker/forked-projects.json:470]: I found https://stackoverflow.com/questions/68088682/how-do-i-stop-githubs-web-editor-from-adding-a-newline-to-the-end-of-file but didn't test anything.

Please remove unnecessary changes so I can accept your PR.

Comment (bbelide2) [reply format_checker/forked-projects.json]: Removed unnecessary changes

**Comment by bbelide2** [comment]
> Please squash commits.

Squashed all the commits and merged new changes

**Comment by darko-marinov** [comment]
@zzjas can you please double check? Things looked fine to me, but the PR is so big that it's good to have two reviewers.

**Comment by bbelide2** [comment]
> Contributor

I added only 2 tests, but the diff is showing all the changes done by commits in between as well. I am not sure why but when I tried to squash, I am getting all the changes made by other people in between as well. Should I close this PR and open a new fresh PR instead?

**Comment by darko-marinov** [comment]
Thanks for cleaning this even while your laptop is broken!
